### ISSUE TITLE
Message ZWED0115E has 2 different meanings

### ISSUE THREAD

**Issue by John-A-Davies** [zowe/zlux-server-framework#194]
Message ZWED0115E occurs in 
```
 zowe/zlux-server-framework
lib/assets/i18n/log/messages_en.json 
```
but also in
```
 zowe/zlux-server-framework
plugins/config/lib/assets/i18n/log/messages_en.json 
```
with a different explanation.  To be fair, the text after `ZWED0115E` differs, so the user knows which one it is, but it can be confusing, all the more confusing if we plan to put descriptions of all message codes in our user documentation.

Also, `Only GET method supported` occurs 6 times in that `plugins` file, with a different message number each time.  This suggests that `ZWED0115E` is not just an event, but a unique site in the code where that event can occur.  I expect users would assume that `ZWED0115E` was an event.

**Comment by John-A-Davies** [comment]
@1000TurquoisePogs Can you please assign this, as I can't do that in this repo.

**Comment by John-A-Davies** [comment]
Fixed already? #192

**Comment by 1000TurquoisePogs** [comment]
Yes, appears to have been fixed, but thank you for making the ticket regardless.
